README.md

```diff
@@ -1,30 +1,34 @@
 linter-flake8
 =============
 
-linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8 provider for [Linter](https://github.com/AtomLinter/Linter).
+linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for [Atom-Linter](https://github.com/AtomLinter/Linter).
```

Comment:
There are lots of `Linter`s, one might confuse this e.g. with Sublime Linter; just wanted to mention we're talking Atom here. Will reformulate to "linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for Atom [Linter](https://github.com/atom-community/linter) package"